editor/src/clj/editor/code/shader.clj

```diff
@@ -140,8 +151,9 @@
         spirv-shader-or-errors-or-nil (when compile-spirv
                                         (make-spirv-shader full-source resource-ext resource-path))]
     (g/precluding-errors spirv-shader-or-errors-or-nil
-      (let [glsl-shader (make-glsl-shader full-source)
-            shaders (filterv some? [glsl-shader spirv-shader-or-errors-or-nil])
+      (let [glsl-shader (make-glsl-shader full-source resource-ext resource-path "glsl")
+            gles2-shader (make-glsl-shader full-source resource-ext resource-path "gles2")
+            shaders (filterv some? [glsl-shader gles2-shader spirv-shader-or-errors-or-nil])
```

Comment:
Since we don't know the platform's gles version, we bundle both versions and the engine will pick the one it wants.